Makefile

```diff
@@ -1,6 +1,6 @@
-GPU=0
-CUDNN=0
-OPENCV=0
+GPU=1
```

Comment:
probably don't want these changes in the main repo for everybody by default

Reply:
You're right! I'll fix that. Thx :)